### PR TITLE
Animations

### PR DESCRIPTION
**Pull request by joacomiguens**
## Summary

- Create FadeIn Animation.

## Screenshot

animations

## Trello Card

You can find it [here](https://trello.com/c/QkD9jezj/36-animations)

### CODE REVIEW

File: scss/application.scss [hunk 1]
```diff
@@ -4,9 +4,7 @@
 @import 'mixins';
 @import 'global-classes';
 @import 'variables';
-@import 'global-classes';
 @import 'navbar';
-@import 'login';
```
Comment (ebedoyaalzate) [inline scss/application.scss:9]: Why you delete this?

Comment (ebedoyaalzate) [inline scss/application.scss:9]: why you delete this

Comment (ebedoyaalzate) [inline scss/application.scss:9]: You need to delete this?

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: scss/application.scss (hunk 1 above)
Comment (joacomiguens) [reply scss/application.scss:9]: Not at all, I fix it now. 😃

File: scss/application.scss [hunk 2]
```diff
@@ -4,7 +4,6 @@
 @import 'mixins';
 @import 'global-classes';
 @import 'variables';
-@import 'global-classes';
```
Comment (ebedoyaalzate) [inline scss/application.scss:7]: also this

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

**Review by ebedoyaalzate — APPROVED** [review]

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**